mm2src/coins/eth.rs

```diff
@@ -762,6 +765,23 @@ struct SavedErc20Events {
     latest_block: U64,
 }
 
+/// Specifies which blockchain the EthCoin operates on: EVM-compatible or TRON.
+/// This distinction allows unified logic for EVM & TRON coins.
+#[derive(Clone, Debug)]
+pub enum ChainSpec {
+    Evm { chain_id: u64 },
+    Tron { network: tron::Network },
```

Comment:
why not [Tvm](https://developers.tron.network/docs/tvm) for `Tron` variant?

Reply:
Let's leave it as Tron until we find another chain that uses Tvm, I don't think there will ever be one :)